src/Mod/Arch/Presets/pset_definitions.csv

```diff
@@ -26,7 +26,7 @@ Pset_AudioVisualApplianceTypeCommon;Reference;IfcIdentifier
 Pset_AudioVisualApplianceTypeDisplay;NominalSize;IfcPositiveLengthMeasure;DisplayWidth;IfcPositiveLengthMeasure;DisplayHeight;IfcPositiveLengthMeasure;Brightness;IfcIlluminanceMeasure;ContrastRatio;IfcPositiveRatioMeasure;RefreshRate;IfcFrequencyMeasure;VideoResolutionWidth;IfcInteger;VideoResolutionHeight;IfcInteger
 Pset_AudioVisualApplianceTypePlayer;PlayerMediaEject;IfcBoolean
 Pset_AudioVisualApplianceTypeProjector;VideoResolutionWidth;IfcInteger;VideoResolutionHeight;IfcInteger
-Pset_AudioVisualApplianceTypeSpeaker;Impedence;IfcFrequencyMeasure
+Pset_AudioVisualApplianceTypeSpeaker;Impedance;IfcFrequencyMeasure

```

Comment:
This shoudn't be changed, it's not a comment, it comes from the IFC format. If we changed it we'll loose compatibility

Reply:
reverted commit. Will try to submit something upstream at some point.